commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt

```diff
@@ -16,3 +16,6 @@ val ExtSubmission.isCollection
 
 val ExtSubmission.filesPath
     get(): String = "$relPath/Files"
+
+val ExtSubmission.description
```

Comment:
I think it's confusing that the description actually returns the title. We should call it something like `submissionTitle` or something like that.

Reply:
I had issues naming this as sometimes is the section title and submissionTitle is kind of redudant as it bellows to submission object and is confusing with the title attribute itself.